public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <link rel="shortcut icon" href="../src/images/abstergo.jpg">
+    <link rel="shortcut icon" href="../src/components/atoms/abstergo.jpg">
```

Comment:
You can't do this. `index.html` transfers to user (to browser) without any transformation. Imagine: you opens `http://localhost:3000/` and browser try to load `../src/components/atoms/abstergo.jpg`. You open root page. There is no ability to get parent directory (`../`).

If you want to control elements in `<head>` section from React see following library: https://github.com/nfl/react-helmet